src/components/Avatar.tsx

```diff
@@ -11,15 +10,16 @@ import {
 
 import type { Theme } from '../types/Theme';
 import { withTheme } from '../';
-import type { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
+import { ViewStyle } from 'react-native';
```

Comment:
It'll still be good to do:

```suggestion
import type { ViewStyle } from 'react-native';
```

We can add `importsNotUsedAsValues": "error",` in `compilerOptions` of tsconfig to enforce this.